src/NVMLIOGroup.hpp

```diff
@@ -80,8 +80,8 @@ namespace geopm
             void register_signal_alias(const std::string &alias_name, const std::string &signal_name);
             void register_control_alias(const std::string &alias_name, const std::string &control_name);
 
-            std::map<pid_t, int> accelerator_process_map(void) const;
-            double cpu_accelerator_affinity(int cpu_idx, std::map<pid_t, int> process_map) const;
+            std::map<pid_t, double> accelerator_process_map(void) const;
```

Comment:
Why did all of these integers become doubles?

Reply:
This change was made to more cleanly pass along NAN for cases of CPUs associated with multiple accelerators, to keep it in line with the expect_same aggregation semantics.